performance_manager/tests/july_17_filepaths.json

```diff
@@ -1,31 +1,4 @@
-#!/usr/bin/env python
```

Comment:
removed this file now that we no longer need it.

i moved the "files to load" list and put it into a dict in tests.

git thinks its a rename and edit, which, lol.